pages/index.js

```diff
@@ -75,50 +79,68 @@ export default function Home({
   return (
     <>
       <Header />
-      <StyledMain>
-        <NewWordForm onCreateNew={pushNewWord} allWords={allWords} />
-        {wordsInCategories.map((item) =>
-          editing && editId === item.categoryName ? (
-            <Fragment key={item.id}>
-              <EditCategory
-                item={item}
-                onReturnFromEditMode={onReturnFromEditMode}
-                onSaveEdited={handleEditedCategory}
-                editId={editId}
-              />
-            </Fragment>
-          ) : (
-            <Fragment key={item.id}>
-              <StyledLink
-                aria-label={`open ${item.categoryName} category`}
-                href={`/${item.categoryName}`}
-              >
-                <StyledCategory>
-                  <h3>{item.categoryName}</h3>
-                  <p>
-                    {item.categoryWords.length}{" "}
-                    {item.number === 1 ? "word" : "words"}
-                  </p>
-                  {item.categoryName === popup ? (
-                    <PopupMenu
-                      onClosePopup={onClosePopup}
-                      id={item.categoryName}
-                      onDelete={handleDeleteCategory}
-                      onEdit={onEdit}
-                      prop={item}
-                    />
-                  ) : (
-                    <PopupMenuButton
-                      id={item.categoryName}
-                      onPopupClick={onPopupClick}
-                    />
-                  )}
-                </StyledCategory>
-              </StyledLink>
-            </Fragment>
-          )
+      <LoginSection>
+        {session ? (
+          <>
+            <div>
+              <p>you are signed in as </p>
+              <p>{session?.user?.name}</p>
+            </div>
+            <BiggerActionButton onClick={signOut}>SIGN OUT</BiggerActionButton>
```

Comment:
is signOut a function from nextAuth that you don't have to write yourself and it just works like magic? :D

Reply:
yes, it's all background magic that you can just import 🙈